build.gradle

```diff
@@ -191,4 +191,4 @@ def readVersion() {
         }
     }
     return version
-}
+}
```

Comment:
newline missing?

Reply:
Did copy and paste work from working release/1.3 branch